ledger-service/jwt/src/test/scala/com/digitalasset/jwt/JwksSpec.scala

```diff
@@ -69,6 +74,47 @@ class JwksSpec extends AnyFlatSpec with Matchers {
       s"The token with a mismatching public key should not successfully verify",
     )
   }
+
+  it should "invalidate cached key-id if verification failed" in new JwksSpec.Scope {
```

Comment:
No tagging?

Reply:
I was not sure what is being exploited in this test in terms of security... do you have an idea?